Explorer/Assets/Scripts/ECS/SceneLifeCycle/LocalSceneDevelopment/LocalSceneDevelopmentController.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Cysharp.Threading.Tasks;
+using DCL.Diagnostics;
+using Decentraland.Sdk.Development;
+using ECS.SceneLifeCycle.Systems;
+using Google.Protobuf;
+using System;
+using System.Net.WebSockets;
+using System.Threading;
+
+namespace ECS.SceneLifeCycle.LocalSceneDevelopment
+{
+    [LogCategory(ReportCategory.SDK_LOCAL_SCENE_DEVELOPMENT)]
```

Comment:
This attribute is only respected by systems via Code generation, there is no magic for other classes unless you handle attributes manually